studio/localStores/storageExplorer/StorageExplorerStore.js

```diff
@@ -97,17 +96,16 @@ class StorageExplorerStore {
     }
   }
 
-  initStore(projectRef, url, serviceKey) {
+  initStore(projectRef, url, serviceKey, protocol = "http") {
```

Comment:
Is it possible to default to https?

Reply:
I set to `http` due to the two commits I linked in the PR template, but setting to `https` should be safe.
Actually that argument should never be undefined in the code and so the default value should never be used.
I added it to not break the API of that function

Reply:
How about setting the default to `PROJECT_ENDPOINT_PROTOCOL` to be consistent with current behaviour if user pass in `undefined`?

```suggestion
  initStore(projectRef, url, serviceKey, protocol = PROJECT_ENDPOINT_PROTOCOL) {
```